src/CarbonAware.LocationSources/src/NamedGeoposition.cs

```diff
@@ -1,4 +1,4 @@
-namespace CarbonAware.Model;
+namespace CarbonAware.LocationSources;
```

Comment:
NIT:  Consider moving this under a `/Models` dir and namespace to be consistent with other projects